src/state/chainstate.rs

```diff
@@ -251,7 +549,34 @@ impl ChainstateManagerBuilder {
         self
     }
 
-    /// Run the chainstate db in-memory only. No database files will be written to disk.
+    /// Configures the chainstate database to run entirely in memory.
+    ///
+    /// When enabled, the chainstate database (which stores the current UTXO set)
+    /// will be stored in RAM rather than on disk. This can significantly improve
+    /// performance but requires substantial memory (several gigabytes for mainnet)
+    /// and means the database will be lost when the process exits.
+    ///
+    /// # Arguments
+    /// * `chainstate_db_in_memory` - If true, run the chainstate database in memory
+    ///
+    /// # Memory Requirements
+    /// The chainstate database size varies by network:
+    /// - Mainnet: ~5-10 GB
+    /// - Testnet: ~1-2 GB
+    /// - Regtest: Typically very small
```

Comment:
I think metrics that are guaranteed to change over time should not be in the docs.